js/webpack.config.js

```diff
@@ -32,16 +33,16 @@ module.exports = [
         entry: './src/index.js',
         output: {
             filename: 'index.js',
-            path: '../firefly_widgets/static',
+            path: path.resolve(__dirname, '..', 'firefly_widgets', 'static'),
             libraryTarget: 'amd'
         },
         devtool: 'source-map',
         module: {
-            loaders: loaders
+            rules: rules
         },
-        externals: ['jupyter-js-widgets']
+        externals: ['@jupyter-widgets/base']
     },
-    {// Embeddable jupyter-firefly bundle
+    {// Embeddable {{ cookiecutter.npm_package_name }} bundle
```

Comment:
I think running the cookie cutter would insert `jupyter-firefly` here -- it is just a comment, I realize.